### PR TITLE
#1445 fix for building options for the VisualizerRuntimeTemplate.

### PR DESCRIPTION
**Pull request by alexpantyukhin**

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
@GraemeBradbury mind taking a look at this?

File: src/core/Akka.MultiNodeTestRunner.Shared/Persistence/VisualizerRuntimeTemplate.Tree.cs [hunk 1]
```diff
@@ -77,31 +77,41 @@ public string BuildOptions(FactData spec)
...
 
-            var firstEventTimeStamp = events.Aggregate(
-                (aggregate, nextValue) =>
-                    aggregate > nextValue
-                        ? nextValue
-                        : aggregate);
+            var startEventTimeParameter = "null";
+            var endEventTimeParameter = "null";
+
+            if (events.Any())
```
Comment (GraemeBradbury) [inline src/core/Akka.MultiNodeTestRunner.Shared/Persistence/VisualizerRuntimeTemplate.Tree.cs]: `.Any()` always allocates an enumerator, when you're just checking for "not empty" then `events.Count > 0` works better.

**Comment by GraemeBradbury** [comment]
Thanks for fixing this. It's appreciated.
:+1:

**Comment by alexpantyukhin** [comment]
Thank you for review!

**Comment by Aaronontheweb** [comment]
Re-running the build on this thing now - intermittent MNTR failure by the looks of it

**Pull request merged**